spec/models/comment_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'rails_helper'
+
+RSpec.describe Comment, type: :model do
+
+  context 'factory tests for projects comment' do
+    subject { build(:comment, :for_project) }
+    it { is_expected.to be_valid }
+  end
+
+  context 'factory tests for tasks comment' do
+    subject { build(:comment, :for_task) }
+    it { is_expected.to be_valid }
```

Comment:
dry this up a bit

Reply:
context 'factories' do
    it { build(:comment, :for_project).to be_valid }
    it { build(:comment, :for_task).to be_valid }
  end

for me something like would be enough